### PR TITLE
ci: bring fatal-meson-warnings back

### PR DESCRIPTION
**Pull request by evverx**
It's a follow-up to https://github.com/systemd/systemd/pull/23204

### CODE REVIEW

**Review by yuwata — APPROVED** [review]
LGTM.

File: .github/workflows/build_test.sh [hunk 1]
```diff
@@ -123,12 +123,14 @@ for args in "${ARGS[@]}"; do
...
-    # break running with older versions
-    # FIXME: re-enable once the minimum version is bumped to 0.60
-    #if [[ "$LINKER" != lld ]]; then
-    #    additional_meson_args="--fatal-meson-warnings"
-    #fi
+
+    # The install_tag feature introduced in 0.60 causes meson to fail with fatal-meson-warnings
+    # "Project targeting '>= 0.53.2' but tried to use feature introduced in '0.60.0': install_tag arg in custom_target"
+    # It can be safely removed from the CI since it isn't actually used anywhere to test anything.
+    find . -type f -name meson.build -print0 | xargs -0 sed -i '/install_tag/d'
```
Comment (medhefgo) [inline .github/workflows/build_test.sh]:
```suggestion
    find . -type f -name meson.build -exec sed -i '/install_tag/d' '{}' '+'
```

File: .github/workflows/unit_tests.sh [hunk 2]
```diff
@@ -60,10 +60,11 @@ for phase in "${PHASES[@]}"; do
...
                 MESON_ARGS+=(-Dman=true)
             fi
-            # We use some features (like install_tag) that were introduced in 0.60, but that don't
-            # break running with older versions
-            # FIXME: re-enable once the minimum version is bumped to 0.60
-            # MESON_ARGS+=(--fatal-meson-warnings)
+            # The install_tag feature introduced in 0.60 causes meson to fail with fatal-meson-warnings
+            # "Project targeting '>= 0.53.2' but tried to use feature introduced in '0.60.0': install_tag arg in custom_target"
+            # It can be safely removed from the CI since it isn't actually used anywhere to test anything.
+            find . -type f -name meson.build -print0 | xargs -0 sed -i '/install_tag/d'
```
Comment (medhefgo) [inline .github/workflows/unit_tests.sh]:
```suggestion
            find . -type f -name meson.build -exec sed -i '/install_tag/d' '{}' '+'
```

File: .github/workflows/unit_tests.sh [hunk 3]
```diff
@@ -83,10 +84,11 @@ for phase in "${PHASES[@]}"; do
...
                 fi
             fi
-            # We use some features (like install_tag) that were introduced in 0.60, but that don't
-            # break running with older versions
-            # FIXME: re-enable once the minimum version is bumped to 0.60
-            # MESON_ARGS+=(--fatal-meson-warnings)
+            # The install_tag feature introduced in 0.60 causes meson to fail with fatal-meson-warnings
+            # "Project targeting '>= 0.53.2' but tried to use feature introduced in '0.60.0': install_tag arg in custom_target"
+            # It can be safely removed from the CI since it isn't actually used anywhere to test anything.
+            find . -type f -name meson.build -print0 | xargs -0 sed -i '/install_tag/d'
```
Comment (medhefgo) [inline .github/workflows/unit_tests.sh]:
```suggestion
            find . -type f -name meson.build -exec sed -i '/install_tag/d' '{}' '+'
```

**Review by medhefgo — COMMENTED** [review]
No need to use xargs for this.

**Pull request merged**